src/ucp/core/ucp_rkey.c

```diff
@@ -154,6 +185,10 @@ void ucp_rkey_destroy(ucp_rkey_h rkey)
     unsigned num_rkeys = ucs_count_one_bits(rkey->pd_map);
     unsigned i;
 
+    if (rkey == &dummy_rkey) {
```

Comment:
probably would be nicer to move line 185 (num_rkeys=...) to after this check, so we won't be counting bits of a dummy rkey